tests/api/hello_consumer/README.md

```diff
@@ -0,0 +1,13 @@
+## I am an addon consumer, apparently.
+
+I'm not even sure what an addon or a CMake is. I know I have CMake, NodeJS, and a C++ compiler installed on this system (and this Ninja thing?), but honestly, I have never once had to touch any of that stuff.
+
+I just import addons from my neighbours and they all seem to "just work" for me. I am a happy consumer! :)
+
+Usually, I import these addons from online package repositories in ZIP or TAR format and they all work perfectly. The only reason I am not doing that today is ,because of something about a ```CMakeJS.cmake``` script not being on the 'master' branch yet! so I have to fetch the addon from next door until then, but I'm told that everything will work exactly the same for me, when that does happen.
```

Comment:
just so we're clear;

If the CMake API *were* on master, then this dep would instead be ```"@vendor/hello": "1.0.0"``` and it will work. I am 100% certain that they can pull that addon from a remote registry and it will build, exactly like it does here.

Reply:
I disagree, you would need to publish `@vendor/hello` to npm to be able to reference it with a version number instead of a relative path.
And that could be done with the current version of that example, with it referencing your github branch.